src/lib/viewers/BaseViewer.js

```diff
@@ -1019,6 +1019,15 @@ class BaseViewer extends EventEmitter {
         return permissions.can_annotate || permissions.can_create_annotations;
     }
 
+    /**
+     * Handles the 'scrolltoannotation' event and calls the annotator scroll method
+     * @param {Object} data
+     * @param {string} data.id Annotation id
+     */
+    handleScrollToAnnotation({ id }) {
+        this.annotator.scrollToAnnotation(id);
```

Comment:
Also thinking that if we emit the location information to the scrollToAnnotation we could use that information in box-annotations instead of looking it up from the store.